README.rdoc

```diff
@@ -1,3 +1,5 @@
+https://codeship.com/projects/d2f141b0-5207-0134-86f7-325cd45b0ee2/status?branch=master
```

Comment:
Admittedly I don't know Rdoc syntax that well, but I'm almost positive this is incorrect. It should look something like below, just for Codeship instead of Travis:

```
{<img src="https://travis-ci.org/OCA/partner-contact.svg?branch=9.0" alt="Build Status" />}[https://travis-ci.org/OCA/partner-contact]
```